lib/http-server.js

```diff
@@ -163,12 +163,12 @@ function HttpServer(options) {
   }));
 
   if (typeof options.proxy === 'string') {
-    var proxy = httpProxy.createProxyServer({});
+    var proxy = httpProxy.createProxyServer({
+      target: options.proxy,
+      changeOrigin: true
+    });
     before.push(function (req, res) {
-      proxy.web(req, res, {
-        target: options.proxy,
-        changeOrigin: true
-      }, function (err, req, res, target) {
+      proxy.web(req, res, {}, function (err, req, res, target) {
```

Comment:
What's different about creating the proxy with the options rather than using the options as before?